lib/renaissance/bids/bid.ex

```diff
@@ -28,4 +29,17 @@ defmodule Renaissance.Bid do
     |> Validators.validate_bidder(:bidder_id)
     |> Validators.validate_open(:amount)
   end
+
+  def highest do
+    highest_bids =
+      from(b in __MODULE__,
```

Comment:
I'm not sure how I feel about using __MODULE__ for the name.

Reply:
@kevbuchanan may be better equipped to address this as I'm not all that familiar with `__MODULE__`.

Reply:
From my understanding tho, `__MODULE__` is doing the same thing as if you were to create a query for the bids table **outside** of bid.ex, which might look something like `from b in Renaissance.Bid,`. But because we're querying the bids table **inside** bid.ex we want to call `__MODULE__` to reference the current module.

Reply:
[`__MODULE__`](https://hexdocs.pm/elixir/Kernel.SpecialForms.html#__MODULE__/0)
> Returns the current module name as an atom or nil otherwise.

Reply:
Yeah, like I get what it does, but...for something like database queries it feels safer to use the model name??

Reply:
Would it be reasonable to think of this as a specific example of a more general discussion about the tradeoffs of hardcoded values (_e.g.,_ readability v.s. updatability)?[\*]

[\*] _disclaimer: based on my minimally-informed understanding of the particulars_

Reply:
^ If so, would it be reasonable to view the issue inconsistency (_i.e.,_ using `__MODULE__` only twice), as opposed to the relative merits?

Reply:
I guess my feeling is that readbility/understanding increases if you don't use module

Reply:
Fair enough. The outcome here is the same either way, so
resolved (?) with commit 943604e4b94167600457a84a1e98d88969ed8df4

Reply:
Fair enough. The outcome here is the same either way, so
resolved (?) with commit 943604e4b94167600457a84a1e98d88969ed8df4